Week-3/submission/Session-09_JS_Basics/JavaScript arrays/script.js

```diff
@@ -0,0 +1,29 @@
+const divideArray = (arr) => {
+  const sortArr = arr.sort((a, b) => a - b);
+  let oddNum = [];
+  let evenNum = [];
+  for (let i = 0; i < sortArr.length; i++) {
+    if (sortArr[i] % 2) {
+      oddNum.push(sortArr[i]);
+    } else {
+      evenNum.push(sortArr[i]);
+    }
+  }
+  console.log("Even Number:");
+  for (let i = 0; i < evenNum.length; i++) {
+    if (evenNum.length !== 0) {
+      console.log(evenNum[i]);
+    } else {
+      console.log("no even number");
```

Comment:
you should output none for both odd and even

Reply:
Fixed the changes. please review.